inventory/sample/group_vars/all.yml

```diff
@@ -27,3 +27,6 @@ metal_lb_controller_tag_version: "v0.12.1"
 
 # metallb ip range for load balancer
 metal_lb_ip_range: "192.168.30.80-192.168.30.90"
+
+# local location for .kube/config change "config" to change the name of the local .kube/config | useful if managing a number of kubernetes clusters and configs.
+local_k3s_config_filename: "config"
```

Comment:
Will this overwrite the existing if you have this default?  Curious because I think many will have a `~/.kube/config` already (like me).

Reply:
Also, lint :)  Line is too long, I might change this in the future but maybe a line break ill fix it.